.eslintrc.json

```diff
@@ -9,11 +9,13 @@
         "@typescript-eslint"
     ],
     "rules": {
-        "@typescript-eslint/class-name-casing": "warn",
```

Comment:
This rule is not available in typescript-eslint anymore and is replaced by naming convention